components/renku_data_services/message_queue/orm.py

```diff
@@ -54,9 +55,15 @@ def load(cls, event: Event) -> "EventORM":
         now_utc = datetime.now(UTC).replace(tzinfo=None)
         return cls(timestamp_utc=now_utc, queue=event.queue, payload=message)
 
-    def dump(self) -> Event[dict[str, Any]]:
+    def dump(self) -> Event:
         """Create an event from the ORM object."""
         message = deepcopy(self.payload)
         if "payload" in message and isinstance(message["payload"], str):
             message["payload"] = base64.b64decode(message["payload"])
         return Event(self.queue, message)
+
+    def get_message_type(self) -> str:
```

Comment:
nit-pick: It may be slightly nicer to return `None` when the message type is not present. But it is really not a big deal. So please feel free to ignore this.

Reply:
Oh sure, that is much better, thank you!